src/pages/index.js

```diff
@@ -15,9 +17,16 @@ export default function Home() {
         <meta name="description" content="Generated by create next app" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
-      </Head>
-      <main className={styles.main}></main>
-      <FooterBar />
+      </Head> 
+      
+      <main>
+        <CustomizableAside />
```

Comment:
@annonymousauthority Please could you counsel us on how to get routing set up?

Right now the engineers are calling their various components in `index.js` for QA and visual purposes; I think it's a good idea for us to start building the routes to these actual pages.

How can we start this? You know say I be NextJS JJC! 😖